pkg/apk/auth/auth.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 // DefaultAuthenticators is a list of authenticators that are used by default.
-var DefaultAuthenticators = multiAuthenticator{EnvAuth{}, CGRAuth{}}
+var DefaultAuthenticators = multiAuthenticator{EnvAuth{}, CGRAuth{}, NoOpAuth{}}
```

Comment:
Why is this necessary? I'd expect this to be the same as just returning early from both existing authenticators.